src/components/Header/SearchHeader.tsx

```diff
@@ -13,39 +9,28 @@ import {
   MenuBarWrapper,
   SearchHeaderWrapper,
 } from './SearchHeader.style';
+import SearchAutocomplete from 'components/Search';
+import { ParentSize } from '@vx/responsive';
+import { Region } from 'common/regions';
+import {
+  getSearchAutocompleteLocations,
+  getFilterLimit,
+} from 'components/Search/utils';
```

Comment:
Maybe there should be exported from `components/Search/index` so you can merge the imports?